services/search/testing/test_search_engine.py

```diff
@@ -19,16 +26,25 @@ def test_double_weights_synonyms():
     """
     result = {1: 4}
 
-    assert result == SearchEngine(synonyms_weight=2).search_query(constants.KEYWORDS_ARTICLE_1,
-                                                                  constants.SYNONYMS_ARTICLE_1)
-
-# I mocked the db endpoint to only be for article 1
-# therefore this test is not yet available without the endpoint
-#
-# def test_multiple_articles():
-#    """
-#    Have the query apply for two articles
-#    """
-#    result = {3: 2, 4: 2}
-#    assert result == SearchEngine(constants.ARTICLES).search_query(constants.KEYWORDS_MULTIPLE,
-#                                                                   constants.SYNONYMS_MULTIPLE)
+    with mock.patch('requests.post', return_value=constants.KEYWORDS_DB_MOCK_1) as mocked:
+        with mock.patch('env.get_db_endpoint', return_value="http://somerealurl.com"):
```

Comment:
it would make the test cleaner to extract the mock URL to a variable and reuse it in the assert